src/Devices/EthernetServer_TC.cpp

```diff
@@ -419,29 +429,58 @@ void EthernetServer_TC::sendHeadersWithSize(uint32_t size) {
   // blank line indicates end of headers
   client.write('\r');
   client.write('\n');
-  state = FINISHED;  // TODO: Why?! This is awkward when we want to send a body next.
 }
 
-// 303 response
-void EthernetServer_TC::sendRedirectHeaders() {
-  static const char response[] PROGMEM =
+void EthernetServer_TC::sendResponse(int code) {
+  static const char response_303[] PROGMEM =
```

Comment:
Is this the appropriate place for these constants? Or would it be better in the switch?

Feel free to disagree with me.

Reply:
I think that keeping the actual text out of the switch statement keeps the switch statement smaller and more explicit.

Reply:
I am ambivalent. I thought this would keep the switch more concise. But I see that when someone wants to edit the text of a header they will need to scroll back and forth to make sure they find the correct one. Or maybe they won't need to scroll because they'll just look for the header they want to edit and they won't need to change the switch at all. ¯\_(ツ)_/¯